docs/en/faq/how-kotlin-jdsl-is-different-from-jooq-and-querydsl.md

```diff
@@ -0,0 +1,34 @@
+# How is Kotlin JDSL different from jOOQ and QueryDSL?
+
+Unlike jOOq and QueryDSL, Kotlin JDSL does not generate a `Metadata Model (code generation) for writing queries.
+
+If you use code generation, you'll have to do the following.
+
+```
+Modify tables / entities →
+Compilation errors due to code and metacode mismatch →
+Temporarily change all code associated with the modified code, e.g. by commenting it out, to make it compile.
+Regenerate code
+Remove the temporary code that was processed to pass compilation and replace it with working code.
+```
+
+In addition, JPQL uses a string-based query writing method, so it can pass compilation but may cause errors when running the web application.
```

Comment:
The context doesn't seem to fit. 

How about adding a sentence that connects the sentences?

"Then you might think about trying JPQL yourself, but that's not a good idea.

Reply:
oh! I definitely think it feels natural because I included that sentence.